pkgs/desktops/gnome-3/extensions/chrome-gnome-shell/default.nix

```diff
@@ -7,24 +7,24 @@ name="chrome-gnome-shell";
     rev = "7d99523e90805cb65027cc2f5f1191a957dcf276";
     sha256 = "0qc34dbhsz5yf4z5bx6py08h561rcxw9928drgk9256g3vnygnbc";
   };
- 
+
  buildInputs = [ gnome3.gnome_shell makeWrapper jq dbus gobjectIntrospection
  python python27Packages.requests python27Packages.pygobject3 wrapGAppsHook];
 
  preConfigure = ''
- mkdir build usr etc
- cd build
- ${cmake}/bin/cmake -DCMAKE_INSTALL_PREFIX=$out/usr -DBUILD_EXTENSION=OFF ../
- substituteInPlace cmake_install.cmake --replace "/etc" "$out/etc"  
+   mkdir build usr etc
+   cd build
+   ${cmake}/bin/cmake -DCMAKE_INSTALL_PREFIX=$out/usr -DBUILD_EXTENSION=OFF ../
+   substituteInPlace cmake_install.cmake --replace "/etc" "$out/etc"
  '';
 
  postInstall = ''
     rm $out/etc/opt/chrome/policies/managed/chrome-gnome-shell.json
     rm $out/etc/chromium/policies/managed/chrome-gnome-shell.json
     wrapProgram $out/usr/bin/chrome-gnome-shell \
-      --prefix PATH '"${dbus}/bin/dbus:$PATH"' \
-      --prefix PATH '"${gnome3.gnome_shell}:$PATH"' \
-      --prefix PYTHONPATH : "$PYTHONPATH" 
+      --prefix PATH : '"${dbus}/bin/dbus:$PATH"' \
+      --prefix PATH : '"${gnome3.gnome_shell}:$PATH"' \
+      --prefix PYTHONPATH : "$PYTHONPATH"
```

Comment:
Ah nice catch. I updated the paths to what I think makes more sense, but AFAICT `chrome-gnome-shell` doesn't work as advertised, even with the fixed paths.